src/main/java/com/textmessenger/service/MessageServiceImpl.java

```diff
@@ -21,16 +23,21 @@
   private final UserRepository userRepository;
   private final DialogRepository dialogRepository;
   private final NotificationService notificationService;
+  private SimpMessagingTemplate simpMessagingTemplate;
+  private final String wsPath = "/queue/messages";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=175&issues=AWXEj19jlgbYnohYqVxJ&open=AWXEj19jlgbYnohYqVxJ)

[ISSUE_KEY]: AWXEj19jlgbYnohYqVxJ